### PR TITLE
Fix broken edit hosting preference test

### PR DESCRIPTION
**Pull request by darrenvong**
This is a bit weird, since if you run the test in interactive mode, it'd say it's failed temporarily and then eventually passes 🤔 
but looks like the main thing that broke it was using `window.scroll` on `EditHostingPreference` since jsdom doesn't support it.

### CODE REVIEW

**Comment by lucaslcode** [comment]
It passes for me when I comment out the markdown editors

File: app/frontend/src/features/profile/edit/EditHostingPreference.test.tsx [hunk 1]
```diff
@@ -1,42 +1,54 @@
...
-    );
+  it("should redirect to Profile route after successful update", async () => {
+    renderPage();
 
     userEvent.click(await screen.findByRole("button", { name: SAVE }));
 
-    expect(await screen.findByTitle(mockProfilePage)).toBeInTheDocument();
-  });
+    expect(await screen.findByTestId("user-profile")).toBeInTheDocument();
+  }, 20000);
```
Comment (darrenvong) [inline app/frontend/src/features/profile/edit/EditHostingPreference.test.tsx:53]: So it looks like it's not to do with the markdown editor. I wonder what makes this take that much longer to run than other tests that this needs an increased timeout value...

Comment (lucaslcode) [reply app/frontend/src/features/profile/edit/EditHostingPreference.test.tsx:53]: Maybe it's just a heavier page, frontend tests take quite a long time in CI

**Review by lucaslcode — APPROVED** [review]

File: app/frontend/src/features/profile/edit/EditHostingPreference.test.tsx (hunk 1 above)
Comment (darrenvong) [reply app/frontend/src/features/profile/edit/EditHostingPreference.test.tsx:53]: Yeah the markdown editor must be more heavy to render...

**Pull request merged**